pkg/datasource/datasource.go

```diff
@@ -43,7 +43,7 @@ func NewZabbixDatasource() *ZabbixDatasource {
 }
 
 // newZabbixDatasourceInstance returns an initialized zabbix datasource instance
-func newZabbixDatasourceInstance(dsSettings backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
```

Comment:
This was a breaking change in the plugin sdk.https://github.com/grafana/grafana-plugin-sdk-go/releases/tag/v0.177.0